contracts/contracts/sbtc-token.clar

```diff
@@ -110,6 +112,22 @@
 	)
 )
 
+(define-public (transfer-many (recipients (list 200 { amount: uint, sender: principal, to: principal, memo: (optional (buff 34)) })))
```

Comment:
Q: Should these be within the `;; sip-010-trait` section since they're not part of the trait?

Reply:
Copy. Moved around.